chain/chain/src/chain.rs

```diff
@@ -996,7 +996,7 @@ impl Chain {
         tries: ShardTries,
         gc_config: &near_chain_configs::GCConfig,
     ) -> Result<(), Error> {
-        let _span = tracing::debug_span!(target: "chain", "clear_data").entered();
+        let _span = tracing::debug_span!(target: "garbage_collection", "clear_data").entered();
```

Comment:
Changing the target here seems a bit controversial, what is your goal here? 
Personally I would rather keep it as chain but add the GC prefix as you've done elsewhere. Not a blocker, more of a preference and I don't think we have any guidelines on this so feel free to keep as is.